internal/common/common.go

```diff
@@ -143,6 +144,32 @@ func NormalizeCPUArchitecture(arch string) string {
 	}
 }
 
+// Return available disks and available disks that meet the minSize requirement
+func GetValidDiskCount(disks []*models.Disk, installationDiskID string, minSize int64) (int64, int64) {
+	var eligibleDisks int64
+	var availableDisks int64
+
+	for _, disk := range disks {
+		if (disk.DriveType == models.DriveTypeSSD || disk.DriveType == models.DriveTypeHDD) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
+			if disk.SizeBytes >= conversions.GbToBytes(minSize) {
+				eligibleDisks++
+			} else {
+				availableDisks++
+			}
+		}
+	}
+	return eligibleDisks, availableDisks
+}
+
+func HasOperator(c *models.Cluster, operatorName string) bool {
+	for _, o := range c.MonitoredOperators {
+		if o.Name == operatorName {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Maybe it should be in internal/cluster/common.go?

Reply:
this can cause circular dependencies, please see [the other related comment](https://github.com/openshift/assisted-service/pull/6037#discussion_r1523797814)